src/telemetry/commandTelemetryProps.ts

```diff
@@ -57,3 +53,16 @@ export interface UpdateImageTelemetryProps extends AzdTelemetryProps, ImageSourc
     revisionMode?: KnownActiveRevisionsMode;
     skippedRegistryCredentialUpdate?: 'true';
 }
+
+export interface IngressTelemetryProps {
```

Comment:
I think it would be good to migrate this to its own file `IngressTelemetryProps.ts`